variables.tf

```diff
@@ -299,6 +299,15 @@ variable hc_path {
   default     = "/"
 }
 
+variable hc_request {
+  description = "Health check, the tcp request."
+  default     = ""
+
+}
+variable hc_response {
+  description = "Health check, the tcp response."
+  default     = ""
```

Comment:
what is going to be the value for this in a tcp health check?

Reply:
hmm.. true. We don't provide this. so I guess we don't need this